pkg/network_light.go

```diff
@@ -0,0 +1,43 @@
+package pkg
+
+import (
+	"context"
+	"net"
+)
+
+//go:generate mkdir -p stubs
+//go:generate zbusc -module netlight -version 0.0.1 -name netlight -package stubs github.com/threefoldtech/zos/pkg+NetworkerLight stubs/network_light_stub.go
+
+// NetworkerLight is the interface for the network light module
+type NetworkerLight interface {
+	Create(name string, privateNet net.IPNet, seed []byte) error
+	Delete(name string) error
+	AttachPrivate(name, id string, vmIp net.IP) (device TapDevice, err error)
+	AttachMycelium(name, id string, seed []byte) (device TapDevice, err error)
+	Detach(id string) error
+	Interfaces(iface string, netns string) (Interfaces, error)
+	AttachZDB(id string) (string, error)
+	ZDBIPs(namespace string) ([]net.IP, error)
+	Namespace(id string) string
+	Ready() error
+	ZOSAddresses(ctx context.Context) <-chan NetlinkAddresses
+}
+
+type TapDevice struct {
+	Name    string
+	Mac     net.HardwareAddr
+	IP      *net.IPNet
+	Gateway *net.IPNet
```

Comment:
I think this should be `Routes` instead so we can also have custom routes not only default GW. It's needed with mycelium for example and in other cases where custom routing is required